### PR TITLE
Add twitter link preview

### PR DESCRIPTION
**Pull request by benthecarman**
Link preview image is this, wasn't sure what else to put

image

### CODE REVIEW

**Comment by hsjoberg** [comment]
Thank you for the pull request @benthecarman.
Code looks good to me, do we have any preview of how it potentially would look like?

Also yes loving the carrot, I was also thinking about having a carrot as the favicon. 😁

**Comment by benthecarman** [comment]
You can test it after it's deployed with this, otherwise I am not sure how to test

https://cards-dev.twitter.com/validator

here is the docs:

https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#started

**Comment by hsjoberg** [comment]
> You can test it after it's deployed with this, otherwise I am not sure how to test

Hehe, yeah that was my issue as well.

File: frontend/app.tsx [hunk 1]
```diff
@@ -29,6 +32,15 @@ export default function App({ Page, pageProps }: { Page: ComponentType<any>; pag
         <meta name="viewport" content="width=device-width" />
         <link rel="stylesheet" href="./style/site.css" />
         <link rel="stylesheet" href="./style/reset.css" />
+
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@hampus_s" />
```
Comment (hsjoberg) [inline frontend/app.tsx:37]: Could you make this as a config instead? You can add it to the `frontend` section of the config.
The prop should probably be optional.

**Review by hsjoberg — CHANGES_REQUESTED** [review]

**Comment by hsjoberg** [comment]
@benthecarman I left a comment, would you mind fixing this?

File: frontend/app.tsx (hunk 1 above)
Comment (benthecarman) [reply frontend/app.tsx]: Added the creator from the config, is that just what you wanted or did you want all the options?

**Comment by hsjoberg** [comment]
Tested ACK. I made some small changes.

Thanks for fixing this. 🙏

**Pull request merged**